python-backend/app/api/now_applications/models/activity_summary/settling_pond.py

```diff
@@ -18,9 +18,10 @@ class SettlingPond(ActivitySummaryBase):
                                     primary_key=True)
 
     proponent_pond_name = db.Column(db.String)
-    is_ponds_exfiltared = db.Column(db.Boolean, nullable=False, default=False)
+    is_ponds_exfiltrated = db.Column(db.Boolean, nullable=False, default=False)
     is_ponds_recycled = db.Column(db.Boolean, nullable=False, default=False)
     is_ponds_discharged = db.Column(db.Boolean, nullable=False, default=False)
 
-    def __repr__(self):
```

Comment:
Did you need to mean to remove this `repr`?

Reply:
No. Added back